src/MainPage.js

```diff
@@ -16,22 +17,29 @@ function MainPage({
 
     return (
         <MainPageStyled>
-            <h1>Bin ich in einem Covid-19 Hotspot?</h1>
-            <Form
-                setUserPlace={setUserPlace}
-                errorMessage={errorMessage}
-                isDataLoading={isDataLoading}
-                setSearchOrigin={setSearchOrigin}
-            />
+            <div className="form-wrapper">
+                <h1>Bin ich in einem Covid-19 Hotspot?</h1>
+                <Form
+                    setUserPlace={setUserPlace}
+                    errorMessage={errorMessage}
+                    isDataLoading={isDataLoading}
+                    setSearchOrigin={setSearchOrigin}
+                />
+            </div>
+
+            <FAQ />
         </MainPageStyled>
     )
 }
 
 const MainPageStyled = styled.div`
-    padding: 10px;
     display: grid;
-    grid-gap: 20px;
-    background: var(--silver);
+    grid-gap: 40px;
```

Comment:
Have you thought about using % or some other more scalable value here in case you want to make a desktop version of the app as well? In case just focusing on mobile, maybe does not make sense but could also adapt the layout more naturally to different phone screen sizes. :)

Reply:
Thanks for your review!
Definitely I will easy adjust responsiveness for other phone sizes and desktop later on.
Just focused on other things first. :)